vernac/classes.ml

```diff
@@ -401,8 +400,7 @@ let context poly l =
       let cst = Declare.declare_constant ~internal:Declare.InternalTacticRequest id decl in
 	match class_of_constr !evars (EConstr.of_constr t) with
 	| Some (rels, ((tc,_), args) as _cl) ->
-	    add_instance (Typeclasses.new_instance tc Hints.empty_hint_info false (*FIXME*)
```

Comment:
Was this FIXME for poly stuff?

Reply:
Good question!

Reply:
I think so, because it appeared in several of these functions.